japan/cncj-202403/README.md

```diff
@@ -23,6 +21,7 @@ NEED TO BE UPDATED.
 | ------------- | ------------- | ------------- | ------------- |
 |<a href="https://github.com/atoato88"><img src="https://avatars.githubusercontent.com/u/748740?s=50"></a>| <a href="https://github.com/atoato88">Akihito Inoh</a> | @akihito-inou(atoato88) | - sig-cluster-lifecycle<br> - Approver of kubernetes-sigs/kubebuilder-declarative-pattern <br> - sig-docs |
 |<a href="https://github.com/nasa9084"><img src="https://avatars.githubusercontent.com/u/11725486?s=50"></a>| <a href="https://github.com/nasa9084">nasa9084</a> | @nasa9084 | - sig-docs<br> - Approver of kubernetes/website/content/ja |
+| | <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |
```

Comment:
@atoato88 Thank you for adding my information. Can it be updated like below to add my icon?
```
|<a href="https://github.com/mkimuram"><img src="https://avatars.githubusercontent.com/u/36450643?s=50"></a>| <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |
```

It should be looked like below:

| Icon | Name | Slack | Focus on, Role.. |
| ------------- | ------------- | ------------- | ------------- |
|<a href="https://github.com/mkimuram"><img src="https://avatars.githubusercontent.com/u/36450643?s=50"></a>| <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |

Reply:
OK, I'll do that!

Reply:
@mkimuram 
Done..but.. the image is big a little bit..
the `s=50` parameter does not working, I guess.

Reply:
Thank you for updating the suggest above!

Reply:
done.